aresponses/main.py

```diff
@@ -90,7 +96,33 @@ def _host_matches(self, match_host):
                    (match_querystring and _text_matches_pattern(path_pattern, path_qs)):
                     path_matched = True
                     if _text_matches_pattern(method_pattern, method.lower()):
+                        if response is PassThrough:
+                            connector = TCPConnector()
+                            connector._resolve_host = partial(self._old_resolver_mock, connector)
+
+                            new_is_ssl = ClientRequest.is_ssl
+                            ClientRequest.is_ssl = self._old_is_ssl
+                            try:
+                                original_request = request.clone(
```

Comment:
what is cloning for? should that be done for all requests?

Reply:
I want to modify the scheme, which is a read-only property. *Call `.clone(scheme=new_scheme)` for setting up the value explicitly*  http://aiohttp.readthedocs.io/en/v3.0.1/web_reference.html#aiohttp.web.BaseRequest.scheme